### ISSUE TITLE
🚀[2019] Large-Margin Softmax Loss for Convolutional Neural Networks

### ISSUE THREAD

**Issue by cfiken** [cfiken/paper-reading#24]
### タイトル
Large-Margin Softmax Loss for Convolutional Neural Networks
### リンク
https://arxiv.org/abs/1612.02295
### 著者
Weiyang Liu, Yandong Wen, Zhiding Yu, Meng Yang
### 投稿日付
ICML 2016
